From c05dfd186ff8e657d6e07885843859d02045c7d0 Mon Sep 17 00:00:00 2001
From: shengfei Xu <xsf@rock-chips.com>
Date: Mon, 25 Nov 2019 11:53:07 +0800
Subject: [PATCH] arm64: dts: rockchip: remove the regulator-boot-on property
 from the OTG_SWITCH for rk3326 board

the OTG_SWITCH supply OTG, it doesn't need to be on When the driver initializes.

Signed-off-by: shengfei Xu <xsf@rock-chips.com>
Change-Id: I44f4fa618f66f18fa552353c425c7baa6831ab0c
---
 arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dtsi             | 1 -
 arch/arm64/boot/dts/rockchip/rk3326-86v-v10.dts                  | 1 -
 arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts        | 1 -
 arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-robot-linux.dts  | 1 -
 .../boot/dts/rockchip/rk3326-evb-lp3-v10-robot-no-gpu-linux.dts  | 1 -
 arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dtsi             | 1 -
 6 files changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dtsi
index 66a0da9c7243..b43d9c0ef7fb 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dtsi
@@ -543,7 +543,6 @@
 			};
 
 			otg_switch: OTG_SWITCH {
-				regulator-boot-on;
 				regulator-name = "otg_switch";
 			};
 		};
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-86v-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-86v-v10.dts
index 29cffb2ec03c..b38af7282356 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-86v-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-86v-v10.dts
@@ -497,7 +497,6 @@
 			};
 
 			otg_switch: OTG_SWITCH {
-				regulator-boot-on;
 				regulator-name = "otg_switch";
 			};
 		};
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts
index a26e16d20d57..c58cad9ba9c9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts
@@ -598,7 +598,6 @@
 			};
 
 			otg_switch: OTG_SWITCH {
-				regulator-boot-on;
 				regulator-name = "otg_switch";
 			};
 		};
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-robot-linux.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-robot-linux.dts
index b4bad88bfa7f..d0bf6c166490 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-robot-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-robot-linux.dts
@@ -361,7 +361,6 @@
 			};
 
 			otg_switch: OTG_SWITCH {
-				regulator-boot-on;
 				regulator-name = "otg_switch";
 			};
 		};
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-robot-no-gpu-linux.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-robot-no-gpu-linux.dts
index 29a01584bcb0..93bf1c515055 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-robot-no-gpu-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-robot-no-gpu-linux.dts
@@ -356,7 +356,6 @@
 			};
 
 			otg_switch: OTG_SWITCH {
-				regulator-boot-on;
 				regulator-name = "otg_switch";
 			};
 		};
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dtsi
index ea298337833e..6fb28754067e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dtsi
@@ -585,7 +585,6 @@
 			};
 
 			otg_switch: OTG_SWITCH {
-				regulator-boot-on;
 				regulator-name = "otg_switch";
 			};
 		};
-- 
2.35.3

